lib/base/Domain.js

```diff
@@ -31,9 +32,10 @@ Domain.prototype.absoluteUrl = function(uri) {
  * @return {Promise} request promise
  */
 Domain.prototype.request = function(opts) {
-  return this.twilio.request(_.assign({}, opts, {
+  const uri = {
     uri: this.absoluteUrl(opts.uri),
-  }));
+  };
+  return this.twilio.request(Object.assign(opts, Object.getPrototypeOf(opts), uri, Object.getPrototypeOf(uri)));
```

Comment:
Is `Object.getPrototypeOf` needed? Maybe just `Object.assign(opts, uri)` is sufficient?